wfl-slack-app/src/wfl-bot.js

```diff
@@ -0,0 +1,31 @@
+var Slackbot = require('slackbots');
+var request = require('request');
+var handleWflRequest = require('./handle-wfl-request')
+
+var orderIdTable = {};
+var menuSelectionIdTable = {};
+
+var bot = new Slackbot({
+    token: process.env.SLACK_BOT_TOKEN,
+    name: "WFL bot"
+});
+
+bot.on('message', function(data) {
+    if (data.text && data.subtype !== 'bot_message') {
+        if (data.text.indexOf('wfl order create') != -1) {
+            handleWflRequest.handleOrderCreateRequest(bot, data, orderIdTable);
+        }
+        else if (data.text.indexOf('wfl order discard') != -1) {
+            handleWflRequest.handleOrderDiscardRequest(bot, data, orderIdTable);
+        }
+        else if (data.text.indexOf('wfl order complete') != -1) {
+            handleWflRequest.handleOrderCompleteRequest(bot, data, orderIdTable);
+        }
+        else if (data.text.indexOf('wfl gimme') != -1) {
+            handleWflRequest.handleGimmeRequest(bot, data, orderIdTable, menuSelectionIdTable)
+        }
+        else if (data.text.indexOf('wfl') != -1) {
```

Comment:
Note that these are not slash commands. They are just normal messages since the bot is listening to the messages in the channel.

Reply:
Using indexOf finds if that string is anywhere in the text.  Is that what you wanted?  Or did you instead want to have the message start with that text?  You could also use `startsWith` if that's what you want.

Reply:
Used startsWith instead